tools/js/tools/patch_states.js

```diff
@@ -0,0 +1,72 @@
+const fs = require('fs')
```

Comment:
could you put this in https://github.com/PrismarineJS/minecraft-jar-extractor ?
I prefer to keep the amount of js in this repo to a minimum (just validation)

Reply:
done https://github.com/PrismarineJS/minecraft-jar-extractor/pull/19